lib/components/user/notification-prefs-pane.js

```diff
@@ -85,13 +223,60 @@ class NotificationPrefsPane extends Component {
             </FormGroup>
           )}
           {notificationChannel === 'sms' && (
-            <FormGroup>
-              <ControlLabel>Enter your phone number for SMS notifications:</ControlLabel>
-              {/* TODO: Add field validation. */}
-              <FormControl onChange={this._handlePhoneNumberChange} type='tel' value={phoneNumber} />
-            </FormGroup>
+            <div>
+              <FormGroup validationState={phoneFieldValidationState}>
+                <ControlLabel>Enter your phone number for SMS notifications:</ControlLabel>
+                {/* TODO: Add field validation. */}
+
+                <FormControl
+                  onChange={this._handlePhoneNumberChange}
+                  // Fake phone number, see https://en.wikipedia.org/wiki/555_(telephone_number)
+                  placeholder='+15555550123'
+                  type='tel'
+                  value={phoneNumber}
+                />
```

Comment:
A style note: this form element should probably have a max width instead of extending the width of the screen.

Reply:
Fixed.